integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -230,16 +328,55 @@ public static void deleteNamespace(String namespace) throws Exception {
   }
 
   /**
-   * create a namespace.
-   *
-   * @param namespace - namespace to create
-   * @throws Exception - if any error occurs
+   * Create a namespace.
+   * 
+   * @param namespace namespace to create
+   * @throws Exception if any error occurs
    */
   public static void createNamespace(String namespace) throws Exception {
     if (!namespace.equalsIgnoreCase("default")) {
-      String command = "kubectl create ns " + namespace;
-      logger.info("Running " + command);
-      TestUtils.exec(command);
+      try {
+        String cmd1 = "kubectl delete ns " + namespace + " --ignore-not-found";
```

Comment:
why delete it ?

Reply:
This is kind of a before-create-delete-first procedure with " --ignore-not-found" to avoid "AlreadyExists" error.